### DISCUSSION TITLE
Documentation site is only showing v1

### DISCUSSION THREAD

**Discussion by aztekrw** [spatie/laravel-model-states#248] [Bugs]
v2 of the documentation has gone.

https://spatie.be/docs/laravel-model-states/v1/01-introduction

**Comment by aztekrw** [comment]
![Screenshot 2024-02-09 114046](https://github.com/spatie/laravel-model-states/assets/4463406/8185863a-8185-48e3-abd3-9f6b4554d3d2)

**Comment by jpfauatwork** [comment]
Thx, that was why I was unable to get it to work.

**Comment by freekmurze** [comment]
Fixed!
